utils_test.py

```diff
@@ -246,3 +247,117 @@ def test_get_hashable_value(self):
     def test_is_valid_language_code(self):
         self.assertTrue(utils.is_valid_language_code('en'))
         self.assertFalse(utils.is_valid_language_code('unknown'))
+
+    def test_memoize_with_args(self):
```

Comment:
Perhaps add some tests for the case of multiple args / multiple kwargs (to verify things like "swapping the args counts as different", "it's OK if multiple args have the same value", etc.)?

Reply:
Done.